res/layout-land/quota_display_layout.xml

```diff
@@ -0,0 +1,125 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="com.owncloud.android.ui.fragment.QuotaDisplayFragment"
+    android:orientation="horizontal">
+
+
+    <RelativeLayout
```

Comment:
Probably a ```FrameLayout``` would be enough here, since no ```layout_xxx``` property ```RelativeLayout```-specific is used by its children.